pysat/instruments/supermag_magnetometer.py

```diff
@@ -812,7 +813,7 @@ def append_ascii_data(file_strings, tag):
     i = ihead + 1
     while i < len(out_lines) - 1:
         idates.append(i)
-        lsplit = re.split('\t+', out_lines[i])
+        lsplit = re.split('\t', out_lines[i])
```

Comment:
Does this still work?  I recall there being an inexact number of tabs between entries.